src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -118,10 +118,10 @@ export const WorkspaceData = (signal) => ({
     );
     return await res.json();
   },
-  importTdr: async (root: string, instanceId: string, snapshotId: string): Promise<Response> => {
+  importTdr: async (root: string, instanceId: string, snapshotUrl: string): Promise<Response> => {
```

Comment:
Isn't this the manifest url, not the snapshot url?
```suggestion
  importTdr: async (root: string, instanceId: string, manifestUrl: string): Promise<Response> => {
```

Also why not preserve the type here and accept `URL` instead of `string`? Seems like it would be better to use a more restrictive type than `string` if possible.

Reply:
Ah yeah I didn't think too hard about that variable name, good point.  And yes, after double-checking what the API is expecting, URL should work.  Thanks!